### ISSUE TITLE
dicom conversion fails

### ISSUE THREAD

**Issue by AHufnagl** [pytrip/pytrip#429]
I tried to convert dicom files to the trip format using dicom2trip but unfortunately it failed and I got the following error message. 

```
ahufnagl@sirius:~/Downloads$ dicom2trip Pat_pat patpat
Traceback (most recent call last):
  File "/usr/local/bin/dicom2trip", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/pytrip/utils/dicom2trip.py", line 57, in main
    c.read_dicom(dcm)
  File "/usr/local/lib/python2.7/dist-packages/pytrip/ctx.py", line 56, in read_dicom
    self._set_header_from_dicom(dcm)
  File "/usr/local/lib/python2.7/dist-packages/pytrip/cube.py", line 791, in _set_header_from_dicom
    self.patient_name = ds.PatientsName
  File "/usr/local/lib/python2.7/dist-packages/pydicom/dataset.py", line 476, in __getattr__
    return super(Dataset, self).__getattribute__(name)
AttributeError: 'FileDataset' object has no attribute 'PatientsName'
ahufnagl@sirius:~/Downloads$ dicom2trip --version
2.0.1
```

**Comment by grzanka** [comment]
Which version of pydicom library are you using ? Could you please type: `pip show pydicom` and `pip show dicom` and `python -V` send me the output.

**Comment by nbassler** [comment]
think this is #368 related

**Comment by AHufnagl** [comment]
The python version is 2.7.13 
When I type ```pip show dicom``` nothing happens.

```
ahufnagl@sirius:~/Downloads$ pip show pydicom
Name: pydicom
Version: 1.0.2
Summary: Pure python package for DICOM medical file reading and writing
Home-page: https://github.com/pydicom/pydicom
Author: Darcy Mason and contributors
Author-email: darcymason@gmail.com
License: MIT
Location: /usr/local/lib/python2.7/dist-packages
Requires: 
```

**Comment by nbassler** [comment]
I can reproduce with test dicoms:
```
bassler@baslup:~/Projects/pytrip$ dicom2trip /local/dicom/eclipse-8.1.20-phantom-breast/Original/ fofofo
Traceback (most recent call last):
  File "/home/bassler/.local/bin/dicom2trip", line 11, in <module>
    sys.exit(main())
  File "/home/bassler/.local/lib/python2.7/site-packages/pytrip/utils/dicom2trip.py", line 57, in main
    c.read_dicom(dcm)
  File "/home/bassler/.local/lib/python2.7/site-packages/pytrip/ctx.py", line 56, in read_dicom
    self._set_header_from_dicom(dcm)
  File "/home/bassler/.local/lib/python2.7/site-packages/pytrip/cube.py", line 791, in _set_header_from_dicom
    self.patient_name = ds.PatientsName
  File "/home/bassler/.local/lib/python2.7/site-packages/pydicom/dataset.py", line 476, in __getattr__
    return super(Dataset, self).__getattribute__(name)
AttributeError: 'FileDataset' object has no attribute 'PatientsName'
```

seem pydicom got rid of the deprecated tags. We better adapt.

**Comment by nbassler** [comment]
fix on the way.

**Comment by nbassler** [comment]
fixed in #430
